scripts/zero-render-mui-demos.mjs

```diff
@@ -99,6 +100,13 @@ ${renders.join('\n')}
     path.join(process.cwd(), `apps/zero-runtime-vite-app/src/pages/material-ui/${args[0]}.tsx`),
     viteFileContent,
   );
+
+  // run prettier on new files
+  exec('pnpm prettier', (err) => {
+    if (err) {
+      console.error(err);
+    }
+  });
```

Comment:
nit to run pretty-quick when demo files are created.

Reply:
Can this instead import and use via `prettier` api before writing the file ?

Reply:
Done.